2019-05-14-armagaddon/heka/reprocess-structured-landfill/hindsight/run/output/telemetry_s3.cfg.tmp

```diff
@@ -0,0 +1,32 @@
+-- -*- lua -*-
+filename = "moz_telemetry_s3.lua"
+
+-- match data outside the affected period or data that isn't activity-stream
+message_matcher = "Timestamp < 1556967600000000000 || Timestamp > 1557572400000000000 || (Fields[uri] !~ '/submit/activity%-stream')"
```

Comment:
You can use the human readable timestamps (but the above appear to be correct) http://mozilla-services.github.io/lua_sandbox/util/message_matcher.html